client.go

```diff
@@ -537,10 +537,8 @@ func (client *Client) update(data *MetadataResponse) ([]string, error) {
 			break
 		case LeaderNotAvailable:
 			toRetry[topic.Name] = true
-			continue
```

Comment:
there's no election to wait for in that case, the partition is just underreplicated